### DISCUSSION TITLE
Mars helicopter badge?

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#67889] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get the Mars helicopter badge?

**Comment by mehulrajdev** [comment, marked as answer]
Developers who contributed to the specific versions of projects and libraries used by NASA’s Jet Propulsion Laboratory to put the Ingenuity helicopter into the Martian atmosphere will receive a new badge on their profile to commemorate their contributions to this historic achievement in planetary exploration.

You cannot get this badge anymore, as it was only given out in 2020.
